src/main/js/main_html5.js

```diff
@@ -1488,24 +1609,63 @@ require("../../../html5-common/js/utils/environment.js");
       }
 
       var buffer = 0;
-      if (event.target.buffered && event.target.buffered.length > 0) {
-        buffer = event.target.buffered.end(0); // in sec;
-      }
-
-      // durationchange event raises the currentTime as a string
-      var resolvedTime = (event && event.target) ? event.target.currentTime : null;
-      if (resolvedTime && (typeof resolvedTime !== "number")) {
-        resolvedTime = Number(resolvedTime);
+      var newCurrentTime = null;
+      var currentLiveTime = 0;
+      var duration = resolveDuration(event.target.duration);
+
+      // Live videos without DVR (i.e. maxTimeShift === 0) are treated as regular
+      // videos for playhead update purposes
+      if (isDvrAvailable()) {
+        var maxTimeShift = getMaxTimeShift();
+        newCurrentTime = currentTimeShift - maxTimeShift;
+        duration = -maxTimeShift;
+        buffer = duration;
+        // [PBW-5863] The skin displays current time a bit differently when dealing
+        // with live video, but we still need to keep track of the actual playhead for analytics purposes
+        currentLiveTime = _video.currentTime;
+      } else {
+        if (_video.buffered && _video.buffered.length > 0) {
+          buffer = _video.buffered.end(0); // in seconds
+        }
+        // Just a precaution for older browsers, this should already be a number
+        newCurrentTime = ensureNumber(_video.currentTime, null);
       }
 
       var seekable = getSafeSeekRange(getSafeSeekableObject());
-      this.controller.notify(eventname,
-                             { "currentTime": resolvedTime,
-                               "duration": resolveDuration(event.target.duration),
-                               "buffer": buffer,
-                               "seekRange": seekable });
+      this.controller.notify(eventname, {
+        currentTime: newCurrentTime,
+        currentLiveTime: currentLiveTime,
```

Comment:
Are there any docs we need to update with this new parameter?

Reply:
Will check to see if we have a wiki with this somewhere. This parameter is already used in BitWrapper.